table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -211,8 +212,12 @@ public SyncStatus sync(JournalContext context) throws IOException {
 
     if (Files.exists(Paths.get(mConfigPath))) {
       ObjectMapper mapper = new ObjectMapper();
-      DbConfig dbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
-      mBypassSet.addAll(dbConfig.getBypassSet());
+      try {
+        mDbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
+      } catch (JsonProcessingException e) {
+        LOG.warn(String.format("Failed to deserialize UDB config file %s, stays unsynced: {}",
```

Comment:
Here swallow the exception, and keep PC continue, is that your expect logic? Maybe we need to throw the exception and exit the sync method.

BTW, if `(Files.exists(Paths.get(mConfigPath)))` return false, we should throw exception too.

For the LOG, it will help us to find the root cause if print the call stack.

A suggestion way is 

```
LOG.warn("Failed to deserialize UDB config file {}, stays unsynced.", mConfigPath, e);
```

Reply:
> swallow the exception, and keep PC continue

Yes, I thought an invalid config file would warrant a warning, but should not block the sync process. But on second thought, maybe it's better that we make it fail and let the user notice the syntax error ASAP.

> if (Files.exists(Paths.get(mConfigPath))) return false, we should throw exception too

I'm not sure about that. `DB_CONFIG_FILE` in `CatalogProperty` doesn't have a default value, so when the user did not specify any config file when attaching the DB (i.e. they don't want any bypassing behaviour), they would be prevented to do sync because of the exception. I suggest we provide an empty config file as the default value for the config entry.

Also, can the user apply a config file after they have attached the DB? I.e. they did not use a config file initially, but want to add one afterwards? If so, how do we handle this situation?

Reply:
> I suggest we provide an empty config file as the default value for the config entry.

On second thought, `DB_CONFIG_FILE` is the path of the config file, while a default empty config file exists only in code and does *not* have a path. So I would suggest we stay the way it is now (i.e. use `mDbConfig = DbConfig.empty();` in constructor, and ignore the missing config file).

Reply:
Resolved in 1244eb7.